packages/react/src/index.js

```diff
@@ -239,12 +239,16 @@ export {
   Stack as unstable_Stack,
   VStack as unstable_VStack,
 } from './components/Stack';
-export { ContainedTab as unstable_ContainedTab } from './components/Tab';
 export { ContainedTabs as unstable_ContainedTabs } from './components/Tabs';
```

Comment:
Yeah, I think it can be. I wasn't sure, there definitely is still some weirdness with what 'contained' tabs should be able to do but 'line' tabs can't but I really don't like `ContainedTabList` as a a name lol so I would want to find a different workaround